libraries/grpc-sdk/src/modules/config/index.ts

```diff
@@ -48,17 +48,6 @@ export class Config extends ConduitModule<typeof ConfigDefinition> {
     });
   }
 
-  updateConfig(config: any, name: string) {
```

Comment:
This function continues to exist in core.proto. Either remove it from there to (and from config manager) or don't delete it from the sdk